cl/webpack.config.js

```diff
@@ -23,6 +24,13 @@ module.exports = {
       name: 'vendor',
       filename: '[name].js',
     },
+    minimize: true,
+    minimizer: [new TerserPlugin({
+        extractComments: false,
+        terserOptions: {
+          compress: true,
+        },
+      })],
```

Comment:
I don’t love this non-default plugin, but this is evidently how you preserve comments. Moving to `LICENSE.txt` (the default imposed by Webpack 5) is something to consider, but, on balance, the added complication of this plugin is worth removing urgency of reconsidering our source attribution approach!